docs/stylesheets/extra.css

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
This file seems to be of no use since
1.  `primary` and `accent` colors are defined in the scheme configuration
2. `extra_css` is not defined

Reply:
You are right. Have removed the file.

Reply:
See carbynestack/carbynestack#10 to track removal of the legacy documentation in cs:cs.